docs/setup-guide/multi-node-dev-cluster.md

```diff
@@ -37,35 +60,81 @@ gcloud compute scp <master-node-instance>:/var/run/kubernetes/client-ca.crt /tmp
 
+```bash
+mkdir -p /tmp/arktos
+scp -i "<private key of keypair of master node>" ubuntu@<master-node-instance>:/var/run/kubernetes/kubelet.kubeconfig /tmp/arktos/kubelet.kubeconfig
+scp -i "<private key of keypair of master node>" ubuntu@<master-node-instance>:/var/run/kubernetes/client-ca.crt /tmp/arktos/client-ca.crt
+```
+
+4. start the worker node and register into cluster
+
+First make sure the following directories in the worker node are empty. If not, clean them up using sudopermission.
+```
+sudo rm -rf /opt/cni/bin/*
+sudo rm -rf /etc/cni/net.d/*
+```
+
+Then at worker node, run following commands:
+```bash
+export ARKTOS_NO_CNI_PREINSTALLED=y
+
+hostname -i  (to get <worker-ip>)
+export KUBELET_IP=<worker-ip>
+
+OR
+export KUBELET_IP=`hostname -i`
+
+echo $KUBELET_IP
+make clean
+./hack/arktos-worker-up.sh
+```
+
+After the script returns, go to master node terminal and run command "./cluster/kubectl.sh get nodes", you should see the work node is displayed and its status should be "Ready".
+
+But when you run command "./cluster/kubectl.sh get all --all-namespaces", you will see new pod of flannel 'pod/kube-flannel-ds-xxxxx' for worker node is not in Running state. If you check the log of this pod 'pod/kube-flannel-ds-xxxxx', you will see the following error.
+
+```bash
+I0803 22:50:37.646013       1 main.go:520] Determining IP address of default interface
+I0803 22:50:37.646394       1 main.go:533] Using interface with name eth0 and address 172.31.2.184
+I0803 22:50:37.646415       1 main.go:550] Defaulting external address to interface address (172.31.2.184)
+W0803 22:50:37.646432       1 client_config.go:608] Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.
+E0803 22:51:07.648172       1 main.go:251] Failed to create SubnetManager: error retrieving pod spec for 'kube-system/kube-flannel-ds-vgftf': Get "https://10.0.0.1:443/api/v1/namespaces/kube-system/pods/kube-flannel-ds-xxxxx": dial tcp 10.0.0.1:443: i/o timeout
+```
+
+5. On worker node, add one more rule of Linux iptables to fix the above issue
+
 At the moment this doc is written, kube-proxy does not support multi-tenancy yet, and it won't be deployed in the cluster. After kube-proxy issue has been fixed (we will allocate resource to cope with it soon), probably we will also need to copy over kube-proxy related secret and artifacts.
 
 As a temporary measure to accommodate Flannel pods to access api server through the kubernetes service IP (the notorious 10.0.0.1) at the absense of kube-proxy, we need to place a bit trick by running below command __at the worker node__, assuming the kupe-apiserver is listening at https://10.138.0.19:6443 (please substitute with the proper value of your own cluster):
+
 ```bash
 sudo iptables -t nat -A OUTPUT -p tcp -d 10.0.0.1 --dport 443 -j DNAT --to-destination 10.138.0.19:6443
+sudo iptables -t nat -L (for verification)
 ```
```

Comment:
after run  these  two command lines on worker node, I still get crashLoopBackOff error for flannel pod:
```
sonyafenge@doctests-worknode:~/go/src/arktos$ ./cluster/kubectl.sh get all --all-namespaces
NAMESPACE     NAME                            HASHKEY               READY   STATUS             RESTARTS   AGE
kube-system   pod/kube-dns-554c5866fc-mwrw2   3876160263807378935   3/3     Running            4          121m
kube-system   pod/kube-flannel-ds-69ltv       9062874555365188537   1/1     Running            0          83m
kube-system   pod/kube-flannel-ds-b6r5c       3793223391731675759   0/1     CrashLoopBackOff   2          5m55s
kube-system   pod/virtlet-g8c46               8235306323003695292   3/3     Running            0          5m44s
kube-system   pod/virtlet-t47tq               5263497349635468423   3/3     Running            0          83m
```